content/css/concepts/borders/terms/border-color/border-color.md

```diff
@@ -42,6 +42,7 @@ h1 {
   border-style: dotted;
 }
 ```
+![css-border-color-example1](https://raw.githubusercontent.com/Codecademy/docs/main/media/css-border-color-example1.png)
```

Comment:
Add more descriptive `alt-text` (text inside the square brackets).

It should be information about the output image.